filebeat/module/logstash/slowlog/_meta/fields.yml

```diff
@@ -4,6 +4,7 @@
     slowlog
   fields:
     - name: message
+      # This should go into log.original? Or is it `message`?
```

Comment:
I think this is never populated. We should remove the field completely.

Reply:
👍 not seeing this being parsed out in the expected log files: https://github.com/elastic/beats/tree/master/filebeat/module/logstash/slowlog/test